### ISSUE TITLE
Autoteleport in Create

### ISSUE THREAD

**Issue by JustLuuuu** [kodadot/nft-gallery#8400]
We have auto teleport when you are creating NFTs and collections. 

Go to create and create a collection/NFT with the ACC where you have enough funds but on the wrong chain.

## Problem
The button mint is showing the text "Not enough funds" even though I'm eligible to teleport. Till I don't click on the button, "Not enough funds," I don't know about the Teleport option and that I could teleport my funds with auto teleport in order to mint. 

<img width="1278" alt="Screenshot 2023-12-04 at 16 33 25" src="https://github.com/kodadot/nft-gallery/assets/104380776/03f8406f-4561-4268-9644-9a15250e50ea">

https://github.com/kodadot/nft-gallery/assets/104380776/d9b9fe6e-e0d8-4182-b1af-7279a93de827

Not sure if you understand when I want. But when someone has 10 DOTs (not bridged) and wants to create a collection on AHP, the text on the button "not enough funds" dont motivate me to continue minting and finding out about the teleport option.

In case there is a teleport option the button should not show "Not enough funds" but some other text. **Teleport to mint**, or just **mint**.

**Comment by exezbcz** [comment]
@shashkovdanil yes, that is how its intended with a confirmation modal + auto teleport

**Comment by exezbcz** [comment]
@shashkovdanil yeah, and enable auto teleport in the confirmation modal
